tools/run-in-docker

```diff
@@ -8,15 +8,15 @@
 
 # Define the list of supported images
 declare -r IMAGES=(
-    ubuntu:{18.04,20.04,22.04,23.04,devel}
-#    debian:{8,9,10,11,unstable} # can't download package index 31.03.2023
-    opensuse/leap:{42,15}
+    ubuntu:{20.04,22.04,23.04,devel}
+    debian:{10,11,unstable}
+    opensuse/leap:15
     registry.suse.com/suse/sle15
-    centos:{6,7,8} # discontinued
+    centos:{7,8} # discontinued
     sl:7           # discontinued
     quay.io/centos/centos:stream{8,9}
     # registry.access.redhat.com/ubi{7,8,9} basic packages like parted are missing
```

Comment:
You can use CentOS Stream instead of UBI images.

Reply:
Yes, line 17 uses the CentOS Stream images, line 18 is more like a not to self to not bother trying those UBIs

Reply:
Oh, I'm blind, sorry!